openfisca_us/parameters/gov/states/wa/tax/income/capital_gains/deductions/charitable/cap.yaml

```diff
@@ -0,0 +1,12 @@
+description: Washington capital gains charitable contribution cap.
+metadata:
+  label: Washington capital gains charitable contribution cap.
```

Comment:
I think we're avoiding full stops for labels? Not a huge issue though, because it's fairly easy to iteratively switch to one format or another.

Reply:
oops you're right, fixed here and in other labels